_summaries/kang2019decoupling_1.md

```diff
@@ -6,14 +6,90 @@ paper: {{paper_tag}}
 author: # Your GitHub id
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
+### What is the core idea? 
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+The introduces a two-fold approach to solving long-tail distribution problem in image recognitions by using representation learning and classification during the training process.
+
+### How is it realized (technically)?
+
+Long-tail distribution has been a common problem in real world recognition tasks because of imbalance in data classes.
```

Comment:
can you illustrate on how they actually decouple the two (representation learning and classification) more clearly?